src/utils/useInstitutionDetails.tsx

```diff
@@ -15,6 +22,8 @@ const useInstitutionDetails = (
     queryKey: [`fetch-institution`, lei],
     queryFn: async (): Promise<InstitutionDetailsApiType> =>
       fetchInstitutionDetails(auth, lei),
+    staleTime: CACHE_TIME, // 10 minutes - Institution details do not change often
```

Comment:
What about for Type of Financial Institution? It'll have to show that change pretty much immediately for all users with access to the institution if someone adds it on the dedicated Type of Financial Institution first page of the filing flow, otherwise we'll get some weird/stale behaviors? Maybe worth being conservative on caching time until MVP?